.github/workflows/code-analysis-lint-test.yaml

```diff
@@ -127,6 +127,8 @@ jobs:
       PINGCTL_PINGONE_WORKER_CLIENT_SECRET: ${{ secrets.PINGCTL_PINGONE_WORKER_CLIENT_SECRET }}
       PINGCTL_PINGONE_REGION: ${{ secrets.PINGCTL_PINGONE_REGION }}
       PINGCTL_PINGONE_WORKER_ENVIRONMENT_ID: ${{ secrets.PINGCTL_PINGONE_WORKER_ENVIRONMENT_ID }}
+      PINGCTL_LOG_LEVEL: ${{ secrets.PINGCTL_LOG_LEVEL }}
```

Comment:
We could just make these variables rather than secrets, so that we can view their current values in the repository config.

Reply:
switched!